SCCAF/__init__.py

```diff
@@ -330,7 +328,7 @@ def self_projection(X,
                              stratify=cell_types, test_size=fraction)  # fraction means test size
     # set the classifier
     if classifier == 'LR':
-        clf = LogisticRegression(random_state=1, penalty=penalty, C=sparsity, multi_class="ovr", solver="liblinear")
+        clf = LogisticRegression(random_state=1, penalty=penalty, C=sparsity, multi_class="ovr", solver=solver)
```

Comment:
@chichaumiau the test is not passing because the results never converges in the test (see ), I presume that it is due to this change. Could you try to run the test locally on your machine and see why they never converge with the new solver default? The test always use the latest version of the tool (so, your changes). If you think that the tests need updating, has the way of calling SCCAF changed? and why? specially if it according to you its only a few changes. 

Tests are currently executing what it is in the .travis file at the root of the repo, take a look at the steps and let me know if they should be changed.

I will re-send the test job to run, but if this is failing every other time it means that there is something unstable in the code.